### PR TITLE
Remove bedgraphtobigwig and add deepTools to rnaseq Docker image

### PR DESCRIPTION
**Pull request by JenkoB**
## Checklist

* [x] Update CHANGELOG.rst for each commit separately:
  * Pay attention to write entries under the "Unreleased" section.
  * Mark all breaking changes as "**BACKWARD INCOMPATIBLE:**" and put them
    before non-breaking changes.
  * If a commit modifies a feature listed under "Unreleased" section,
    it might be sufficient to modify the existing CHANGELOG entry from previous
    commit(s).
* [ ] Bump the process version:
  * **MAJOR version (first number)**: Backward incompatible changes (changes
    that break the api/interface). Examples: renaming the input/output, adding
    mandatory input, removing input/output...
  * **MINOR version (middle number)**: add functionality or changes in a
    backwards-compatible manner. Examples: add output field, add non-mandatory
    input parameter, use a different tool that produces same results...
  * **PATCH version (last number)**: changes/bug fixes that do not affect
    the api/interface. Examples: typo fix, change/add warning messages...
* [ ] All inputs are used in process.
* [ ] All output fields have their ``re-save`` calls.

### CODE REVIEW

**Comment by JenkoB** [comment]
We will use deepTools package instead of bedtools with BedgraphToBigwig way for calculating BigWig from Bam files. Because  bedgraphtobigwig is not needed I have removed it from rnaseq Docker image.
@JureZmrzlikar @mzganec please review

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -128,6 +128,8 @@ Changed
 
   - ``numpy`` (v1.14.4)
   - ``pandas`` (v0.23.0)
+  - remove ``bedgraphtobigwig`` and add ``deepTools`` to ``resolwebio/rnaseq:3.4.0``
```
Comment (mzganec) [inline docs/CHANGELOG.rst]: Wrong indentation. Add `deepTools` version. Consider establishing a relationship between the two tools, e.g. _replace `bedgraphtobigwig` with `deepTools`_ and provide a brief explanation why this is needed.

File: resolwe_bio/docker_images/rnaseq/packages-debian.txt [hunk 2]
```diff
@@ -25,3 +25,6 @@ libssl-dev
 python3-numpy
 python3-pandas
 python3-sklearn
+python3-matplotlib
```
Comment (mzganec) [inline resolwe_bio/docker_images/rnaseq/packages-debian.txt]: Order packages alphabetically.

File: resolwe_bio/docker_images/rnaseq/README.md [hunk 3]
```diff
@@ -10,7 +10,6 @@ Included bioinformatics tools:
 * Bedtools (2.27.1)
 * Bowtie (v1.2.2)
 * Bowtie2 (v2.3.4.1)
-* bedGraphToBigWig (kent-v365)
```
Comment (mzganec) [inline resolwe_bio/docker_images/rnaseq/README.md]: Add `deepTools`.

**Pull request merged**